concourse/setup.py

```diff
@@ -196,7 +196,7 @@ def create_instance_specific_helm_values(concourse_cfg: ConcourseConfig):
                 }
             }
         }
-    elif concourse_version == "4":
+    elif concourse_version == 4:
```

Comment:
so you changed from str to int. in this case: please already do this in the first commit ;-)

also: please replace with enum (as commented above)

Reply:
done